### DISCUSSION TITLE
Postgre Immortal MR resolving

### DISCUSSION THREAD

**Discussion by DerHerrGammler** [renovatebot/renovate#20607] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

Gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hey,

I currently have a MR named `chore(deps): update postgres docker tag` that updates ALL the postgre docker tags in my docker-compose files to the latest version:

| Package | Update | Change |
|---|---|---|
| postgres | minor | `15.1-alpine` -> `15.2-alpine` |
| postgres | major | `11-alpine` -> `15-alpine` |
| postgres | major | `12.11` -> `15.2` |

But i want them split in different MR so i can update one postgres server at a time.

My current Renovate settings are pretty basic and most ist extended from base:
```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": ["config:base"],
    "labels": ["dependencies"],
    "packageRules": [
        {
            "groupName": "html-eslint monorepo",
            "matchPackagePatterns": ["^@html-eslint/"]
        }
    ],
    "assignAutomerge": true,
    "assignees": [],
    "assigneesFromCodeOwners": false,
    "assigneesSampleSize": 1,
    "reviewers": ["derherrgammler"],
    "reviewersFromCodeOwners": false,
    "reviewersSampleSize": 1,
    "baseBranches": ["dev"],
    "branchConcurrentLimit": 0,
    "prConcurrentLimit": 0,
    "prHourlyLimit": 0,
    "branchPrefix": "renovate/",
    "commitBodyTable": true,
    "draftPR": false,
    "rebaseWhen": "conflicted",
    "rangeStrategy": "pin",
    "rollbackPrs": true,
    "dependencyDashboard": true,
    "dependencyDashboardAutoclose": true,
    "dependencyDashboardLabels": ["dependencies", "dev | development"],
    "separateMajorMinor": true
}
```

What packageRules do i have to add so that i can achive what i need to achive so that the MR is split in multiple and they are no longer immortal.

### What you tried so far.

I played around with the packageRules for matchUpdateTypes but i can't figure out what am i wrong.

**Comment by viceice** [comment, marked as answer]
additional branch prefix 😉

**Reply by DerHerrGammler** [reply]
Can i configure the additional branch prefix to apply only on docker/docker-compose related packages?

**Reply by secustor** [reply]
Yes, look into packageRules with `matchManagers`.

**Reply by DerHerrGammler** [reply]
thanks that helped me
